scripts/skills/injury/rf_black_eye_injury.nut

```diff
@@ -0,0 +1,75 @@
+this.rf_black_eye_injury <- this.inherit("scripts/skills/injury/injury", {
+	m = {
+		VisionModifier = -1,
+		MeleeSkillMult = 0.9,
+		RangedSkillMult = 0.9,
+	},
+	function create()
+	{
+		this.injury.create();
+		this.m.ID = "injury.rf_black_eye";
+		this.m.Name = "Black Eye";
+		this.m.Description = "A powerful blow to the eye socket left the area bruised, swollen, and discolored.";
+		this.m.Type = this.m.Type | ::Const.SkillType.TemporaryInjury;
+		this.m.DropIcon = "rf_black_eye_injury";
+		this.m.Icon = "ui/injury/rf_black_eye_injury.png";
+		this.m.IconMini = "rf_black_eye_injury_mini";
+		this.m.HealingTimeMin = 1;
+		this.m.HealingTimeMax = 3;
+	}
+
+	function getTooltip()
+	{
+		local tooltip = this.skill.getTooltip();
+
+		if (this.m.VisionModifier != 0)
+		{
+			tooltip.push({
+				id = 7,
```

Comment:
I've been starting our ids at 10 (as it gives more room for other mods who may be adding between 7-9). What do you think?